src/test/java/kitchenpos/application/OrderTableServiceIntegrationTest.java

```diff
@@ -0,0 +1,180 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.ThrowableAssert;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import kitchenpos.IntegrationTest;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuGroupRepository;
+import kitchenpos.domain.MenuRepository;
+import kitchenpos.domain.OrderRepository;
+import kitchenpos.domain.OrderTable;
+import kitchenpos.domain.OrderTableRepository;
+import kitchenpos.domain.Product;
+import kitchenpos.domain.ProductRepository;
+import kitchenpos.fixture.MenuFixture;
+import kitchenpos.fixture.MenuGroupFixture;
+import kitchenpos.fixture.OrderFixture;
+import kitchenpos.fixture.OrderTableFixture;
+import kitchenpos.fixture.ProductFixture;
+
+public class OrderTableServiceIntegrationTest extends IntegrationTest {
+	@Autowired
+	private OrderTableService orderTableService;
+	@Autowired
+	private MenuRepository menuRepository;
+	@Autowired
+	private MenuGroupRepository menuGroupRepository;
+	@Autowired
+	private ProductRepository productRepository;
+	@Autowired
+	private OrderTableRepository orderTableRepository;
+	@Autowired
+	private OrderRepository orderRepository;
+
+	@DisplayName("주문 테이블 생성")
+	@Test
+	void 주문_테이블_생성() {
+		// given
+		OrderTable request = new OrderTable();
+		request.setName("9번");
+
+		// when
+		OrderTable orderTable = orderTableService.create(request);
+
+		// then
+		assertThat(orderTable.getId()).isNotNull();
+		assertThat(orderTable.getName()).isEqualTo("9번");
+		assertThat(orderTable.getNumberOfGuests()).isEqualTo(0);
+		assertThat(orderTable.isEmpty()).isEqualTo(true);
+	}
+
+	@DisplayName("주문 테이블 생성 실패 : 이름 없음")
+	@Test
+	void 주문_테이블_생성_실패_1() {
+		// given
+		OrderTable request = new OrderTable();
+		request.setName("");
+
+		// when
+		ThrowableAssert.ThrowingCallable throwingCallable = () -> orderTableService.create(request);
+
+		// then
+		Assertions.assertThatIllegalArgumentException().isThrownBy(throwingCallable);
+	}
+
+	@DisplayName("주문 테이블 착석")
+	@Test
+	void 주문_테이블_착석() {
+		// given
+		OrderTable given = orderTableRepository.save(OrderTableFixture.orderTable());
+
+		// when
+		OrderTable actual = orderTableService.sit(given.getId());
+
+		// then
+		assertThat(actual.getId()).isEqualTo(given.getId());
+		assertThat(actual.isEmpty()).isEqualTo(false);
```

Comment:
```suggestion
		assertThat(actual.isEmpty()).isFalse();
```
assertThat에서 제공하는 인터페이스를 활용해보세요 😄